### PR TITLE
Sorted all areas except Brabant in groups. Brabant is done by Michiel

### PR DESCRIPTION
**Pull request by MartLubben**

### CODE REVIEW

**Comment by MartLubben** [comment]
https://github.com/quintel/etsource/pull/1624

File: datasets/roosendaal/time_curves/energy_extraction_uranium_oxide_time_curve.csv [hunk 1]
```diff
@@ -1,42 +0,0 @@
-year,preset_demand
```
Comment (grdw) [inline datasets/roosendaal/time_curves/energy_extraction_uranium_oxide_time_curve.csv]: Is there a reason these files are removed?

Comment (MartLubben) [reply datasets/roosendaal/time_curves/energy_extraction_uranium_oxide_time_curve.csv]: Yes Roosendaal is double

Comment (MartLubben) [reply datasets/roosendaal/time_curves/energy_extraction_uranium_oxide_time_curve.csv]: So I left the bd_roosendaal and removed the other

**Comment by grdw** [comment]
screen shot 2018-03-21 at 14 55 02

'Brazil' should be in the 'Third party and other regions' group. Much as it is on beta:

screen shot 2018-03-21 at 14 55 33

The same for EU-27

File: datasets/nl2013/nl2013.full.ad [hunk 2]
```diff
@@ -1,4 +1,5 @@
 - area = nl2013
+- group = country
 - id = 1
 - parent_id = 1
 - enabled.etengine = true
```
Comment (grdw) [inline datasets/nl2013/nl2013.full.ad]: One line down there's a `enabled.etmodel` which is on true. This should be false I guess.

File: datasets/roosendaal/time_curves/energy_extraction_uranium_oxide_time_curve.csv (hunk 1 above)
Comment (grdw) [reply datasets/roosendaal/time_curves/energy_extraction_uranium_oxide_time_curve.csv]: Cool! 👍 would love it, if these sort of things can be done in a separate pull request in the future, but I'm fine with this for now.

**Comment by MartLubben** [comment]
Did the requests with this pull request: https://github.com/quintel/etsource/pull/1628

**Pull request merged**